datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -654,7 +666,7 @@ async fn spill_sorted_batches(
     batches: Vec<RecordBatch>,
     path: &Path,
     schema: SchemaRef,
-) -> Result<()> {
+) -> Result<u64> {
```

Comment:
Can we possibly add a brief comment describing that the returned value is the row counts that is spilled?

Reply:
Thanks for the review. Addressed.